webpack/serve.js

```diff
@@ -54,11 +54,20 @@ module.exports = function serve(stripesConfig, options) {
     const port = options.port || process.env.STRIPES_PORT || 3000;
     const host = options.host || process.env.STRIPES_HOST || 'localhost';
 
-    app.use(express.static(`${serverRoot}/public`));
+    const staticFileMiddleware = express.static(`${serverRoot}/public`);
+
+    app.use(staticFileMiddleware);
 
     // Process index rewrite before webpack-dev-middleware
     // to respond with webpack's dist copy of index.html
-    app.use(connectHistoryApiFallback({}));
+    app.use(connectHistoryApiFallback({
+      disableDotRule: true,
```

Comment:
Why are we disabling the dot rule?  It is not clear to me from this PR or the related JIRA ticket what the dev server change accomplishes.

Reply:
Link to this page is consist of path and token. And token usually contains dot in its body(it is normal behavior).  So I decided to disable dot rule to handle this url.

Reply:
Are you saying the token is passed in the URL somewhere?  Could you provide an example request/response for this?